src/components/Shared/SendModal/SendLN/SendLN.tsx

```diff
@@ -7,24 +14,77 @@ const SendLn: FC<SendLnProps> = (props) => {
   const { balance, onChangeInvoice, onConfirm } = props;
   const { t } = useTranslation();
 
+  const invoiceInput = useRef<HTMLInputElement>(null);
+  const [isFormValid, setIsFormValid] = useState(true);
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const validateInput = (validity: HTMLInputElement["validity"]) => {
+    setIsFormValid(validity.valid);
+    setErrorMessage("");
+
+    if (validity.valueMissing) {
+      setErrorMessage(t("forms.validation.lnInvoice.required"));
+      return false;
+    }
+
+    if (validity.patternMismatch) {
+      setErrorMessage(t("forms.validation.lnInvoice.patternMismatch"));
+      return false;
+    }
+
+    return true;
+  };
+
+  const onChange = (event: ChangeEvent<HTMLInputElement>) => {
+    validateInput(event.target.validity);
+    onChangeInvoice(event);
+  };
+
+  const onSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    const validity = invoiceInput?.current?.validity;
+
+    if (validity) {
+      const isFormValid = validateInput(validity);
+
+      if (isFormValid) {
+        onConfirm(event);
+      }
+    }
+  };
+
   return (
-    <form onSubmit={onConfirm}>
+    <form onSubmit={onSubmit} noValidate>
       <h3 className="text-xl font-bold">{t("wallet.send_lightning")}</h3>
-      <div className="my-5">
+
+      <p className="my-5">
         <span className="font-bold">{t("wallet.balance")}:&nbsp;</span>
         {balance} {appCtx.unit}
-      </div>
+      </p>
+
       <label className="label-underline" htmlFor="invoiceInput">
         {t("wallet.invoice")}
       </label>
+
       <input
         id="invoiceInput"
+        ref={invoiceInput}
         type="text"
-        onChange={onChangeInvoice}
-        className="input-underline"
+        onChange={onChange}
+        required
+        pattern="(lnbc|lntb|lntbs|lnbcrt)\w+"
```

Comment:
`lntbs` and `lnbcrt` can be cut, since they are included in `lnbc` resp. `lntb` and thus if I type `lntbx` that would be valid too.